wgpu-core/src/command/bundle.rs

```diff
@@ -680,7 +680,7 @@ where
 #[derive(Clone, Debug, Error)]
 #[error("{scope}")]
 pub struct RenderBundleError {
-    scope: PassErrorScope,
+    pub scope: PassErrorScope,
```

Comment:
why does it need to be public?

Reply:
I need to access it from the wgpu-rs side, as it is information kind of in the side, with the Ids that need to be resolved